turn-by-turn/api-reference.md

```diff
@@ -127,6 +128,16 @@ All of the options described above for autos also apply to motor_scooter costing
 | `use_primary` | A riders's propensity to use primary roads. This is a range of values from 0 to 1, where 0 attempts to avoid primary roads, and 1 indicates the rider is more comfortable riding on primary roads. Based on the `use_primary` factor, roads with certain classifications and higher speeds are penalized in an attempt to avoid them when finding the best path. The default value is 0.5. |
 | `use_hills` | A riders's desire to tackle hills in their routes. This is a range of values from 0 to 1, where 0 attempts to avoid hills and steep grades even if it means a longer (time and distance) path, while 1 indicates the rider does not fear hills and steeper grades. Based on the `use_hills` factor, penalties are applied to roads based on elevation change and grade. These penalties help the path avoid hilly roads in favor of flatter roads or less steep grades where available. Note that it is not always possible to find alternate paths to avoid hills (for example when route locations are in mountainous areas). The default value is 0.5. |
 
+##### Motorcycle costing options -> **BETA**
+Standard costing for travel by motorcycle.  By default, motorcycle costing will default to higher class roads.  The costing model recognizes factors unique to motorcycle travel and offers options for tuning motorcycle routes. Factors unique to travel by motorcycle influence the resulting route.
+
+All of the options described above for autos also apply to motorcycle costing methods.  These additional options are available for motorcycle costing methods.
```

Comment:
Annoying copy edit: perhaps "The following options are available for motorcycle costing:"?